test_vm/vendor/cookbooks/dbfit_test/recipes/selinux.rb

```diff
@@ -0,0 +1,9 @@
+selinux_state 'SELinux Permissive Temp' do
+  temporary true
+  action :permissive
+end
+
+selinux_state 'SELinux Permissive Perm' do
```

Comment:
If we come up with a good idea - we may try to get s support into selinux cookbook itself (e.g. scope=temporary|configuration-file|both.